docs/tools/mkdocs.md

```diff
@@ -0,0 +1,25 @@
+# MkDocs
+
+We are using MkDocs to generate our documentation. See [Docker-mkdocs repo](https://hackforla.github.io/docker-mkdocs/) for information about MkDocs and the image we're using.
+
+#### Work on docs locally
+
+1. It should already be running with the peopledepot django server. If not, run the mkdocs container.
```

Comment:
Thanks. I'll work on this. I was trying to keep it quick and simple in our docs and people can go to the docs in the link to get more details.

The mkdocs image is a ~40MB download. We don't need to build it. It might take some time depending on connection quality.